package.json

```diff
@@ -4,11 +4,11 @@
   "description": "Collection of general React components used in buildo projects.",
   "main": "lib",
   "scripts": {
-    "test": "node ./depsShouldBePinned.js && jest",
+    "test": "jest",
     "test-dev": "npm test -- --watch",
     "build": "rm -rf lib && mkdir lib && node-sass src --importer sass-importer.js --include-path node_modules -o lib && tsc",
     "preversion": "npm run prettier-check && npm run test && npm run dtslint",
-    "prepublish": "npm run build",
+    "prepublishOnly": "npm run build",
```

Comment:
from npm@5 (and yarn as well, not sure which version) onward, this is correctly run only before running `npm publish`, and NOT after running `npm install` or `yarn`